   ml_ops/sm-datazone_import/README.md

```diff
@@ -31,11 +32,43 @@ Run the script and follow the instructions.
 ```bash
 python import-sagemaker-domain.py \
     --region REGION \
-    --federation-role ARN_OF_FEDERATION_ROLE \
     --account-id ACCOUNTID
 ```
 
 ### Additional Configuration
 
 - SageMaker execution roles need DataZone API permissions in order for the Assets UI to function. See [DataZoneUserPolicy.json](./resources/DataZoneUserPolicy.json) for an example.
-- Ensure the DataZone Domain trusts SageMaker. In the AWS DataZone console navigate to Domain details and select the "Trusted services".
\ No newline at end of file
+- Ensure the DataZone Domain trusts SageMaker. In the AWS DataZone console navigate to Domain details and select the "Trusted services".
+
+### Potential errors and workarounds
+
+**Cannot view ML assets in SageMaker Studio**
+
+Make sure that the execution role that is attached to the SageMaker User in the attached domain has ListTags attached as a permissions policy to the role. A simple workaround is to attach AmazonSageMakerCanvasFullAccess policy which contains this permission. Without it - you will not be able to view the Assets tab in the Studio UI. Expected error in the UI inspect: 
+```
+User: arn:aws:sts::789706018617:assumed-role/AmazonSageMaker-ExecutionRole-20241127T120959/SageMaker is not authorized to 
+perform: sagemaker:ListTags on resource: arn:aws:sagemaker:us-east-1:789706018617:domain/d-qy9jzu4s7q0y because no 
+identity-based policy allows the sagemaker:ListTags action
```

Comment:
well this is what a debugging user will see in network tab of dev tools but user will actually just not see the "Assets" tab on left nav - should describe that before remedy.

Reply:
Sounds good.